model/profile/registry.yaml

```diff
@@ -78,3 +78,38 @@ groups:
                 [Rust](https://wikipedia.org/wiki/Rust_(programming_language))
               value: "rust"
               stability: development
+  - id: registry.profile.pprof
+    type: attribute_group
+    display_name: Pprof specific attributes for the Profiling signal
+    brief: >
+      Attributes specific to pprof that help convert from pprof to Profiling signal.
+    attributes:
+      - id: profile.pprof.mapping.has_functions
```

Comment:
any reason to prefix with`profile` ? 

Following 

https://github.com/open-telemetry/semantic-conventions/blob/1cf2476ae5e518225a766990a28a6d5602bd5a30/docs/general/naming.md?plain=1#L95-L97

I'd suggest `pprof.mapping.*` for this and other attributes.

Reply:
As the brief introduction says, these attributes exist to help converting pprof to Profiling signal and vice versa and so are directly related on how the protocol should be handled. In the scope of other signals, like logs, traces, events or metrics, these attributes are not applicable.

@lmolkova do you still prefer to use `pprof.*` over `profile.pprof.*`?

Reply:
SemConv SIG meeting notes: profle prefix is not necessary - we don't include signals into attribute name.

Reply:
updated the namespace with https://github.com/open-telemetry/semantic-conventions/pull/2522/commits/aab31a24cedb5c5d8de594af9d4f61e3c60e1062.